mms/benchmarks/benchmark.py

```diff
@@ -328,7 +333,7 @@ def decorate_metrics(data_frame, row_to_read):
         if key in AGGREGATE_REPORT_CSV_LABELS_MAP:
             new_key = '{}_{}_{}'.format(benchmark_name, benchmark_model, AGGREGATE_REPORT_CSV_LABELS_MAP[key])
             result[new_key] = value
-
+    result['column'] = row_to_read.replace(' ', '_')
```

Comment:
Were you including the row name as a parameter in this PR as well?

Reply:
This is the row name only. I will change the param name.